hangman.py

```diff
@@ -0,0 +1,300 @@
+import random
+import string
+
+INITIAL_TRIES = 6
+INITIAL_WARN = 3
+WORDLIST_FILENAME = "words.txt"
+
+
+def load_words():
+    """
+    Returns a list of valid words. Words are strings of lowercase letters.
+    Depending on the size of the word list, this function may
+    take a while to finish.
+    """
+    print("Loading word list from file...")
+    inFile = open(WORDLIST_FILENAME, 'r')
+    line = inFile.readline()
+    wordlist = line.split()
+    print("  ", len(wordlist), "words loaded.")
+    return wordlist
+
+
+def choose_word(wordlist):
+    """
+    wordlist (list): list of words (strings)
+    Returns a word from wordlist at random
+    """
+    return random.choice(wordlist)
+
+
+wordlist = load_words()
+
+
+def is_word_guessed(secret_word, letters_guessed):
+    """
+    secret_word: string, the word the user is guessing; assumes all letters are
+      lowercase
+    letters_guessed: set (of letters), which letters have been guessed so far;
+      assumes that all letters are lowercase
+    returns: boolean, True if all the letters of secret_word are in letters_guessed;
+      False otherwise
+    """
+    for letter in secret_word:
+        if letter not in letters_guessed:
+            return True
+    return False
+
+
+def get_guessed_word(secret_word, letters_guessed):
+    """
+    secret_word: string, the word the user is guessing
+    letters_guessed: set (of letters), which letters have been guessed so far
+    returns: string, comprised of letters, underscores (_), and spaces that represents
+      which letters in secret_word have been guessed so far.
+    """
+    compared_list = []
+    for i in secret_word:
+        if i in letters_guessed:
+            compared_list.append(i)
+        elif i not in letters_guessed:
+            compared_list.append("_ ")
+    return "".join(compared_list)
+
+
+def get_available_letters(letters_guessed):
+    """
+    letters_guessed: set (of letters), which letters have been guessed so far
+    returns: string (of letters), comprised of letters that represents which letters have not
+      yet been guessed.
+    """
+    alphabets = set(string.ascii_lowercase)
+    compared_list = [i for i in alphabets if i not in letters_guessed]
+    return "".join(compared_list)
+
+
+def count(warn, tries):
+    """
+    warn: integer, amount of warnings
+    tries: integer, amount of tries
+    This function deletes one warning or a try if there isn't any tries.
+    returns: tuple with amount of warnings and tries.
+    """
+    if warn > 0:
+        warn -= 1
+    else:
+        warn -= 1
+        tries -= 1
+    return warn, tries
+
+
+def vowel_check(letter, tries):
+    """
+        letter: string, entered letter
+        warn: integer, amount of warnings
+        tries: integer, amount of tries
+        This function deletes two tries if entered letter is a vowel, or one if not.
+        returns: amount of tries.
+    """
+    vowels = {"a", "e", "i", "o", "u"}
```

Comment:
This variable is never changed, so it makes sense to declare it as a constant in the top of file